libs/ui/layout/src/lib/download-entry/download-entry.component.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Component,
+  OnInit,
+  ChangeDetectionStrategy,
+  Input,
+} from '@angular/core'
+import { ColorService } from '@geonetwork-ui/util/shared'
+
+@Component({
+  selector: 'gn-ui-download-entry',
+  templateUrl: './download-entry.component.html',
+  styleUrls: ['./download-entry.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DownloadEntryComponent implements OnInit {
+  @Input() format: string
+  @Input() resourceName: string
+  @Input() title: string
+  @Input() description: string
+  @Input() url: string
```

Comment:
missing the ouput on the click + action in the story (actions illustrates the outuput)